lib/libzfs/libzfs_util.c

```diff
@@ -911,16 +925,24 @@ libzfs_envvar_is_set(char *envvar)
 static int
 libzfs_load_module(const char *module)
 {
+#ifdef __FreeBSD__
+
+	if (modfind(module) < 0) {
+		/* Not present in kernel, try loading it. */
+		if (kldload(module) < 0 || modfind(module) < 0) {
+			if (errno != EEXIST)
+				return (-1);
```

Comment:
shouldn't we return errno here, rather than -1?

Reply:
Thanks for catching that, and it made me think twice about the error messages, too.